### PR TITLE
Add option to configure task check interval 

### PR DESCRIPTION
**Pull request by krhubert**
Closes

https://github.com/hibiken/asynq/issues/506
https://github.com/hibiken/asynq/issues/548

### CODE REVIEW

**Review by enzshew — APPROVED** [review]

**Comment by lavrd** [comment]
@krhubert can we merge it? Looks like PR is approved and changes are very helpful.

**Comment by krhubert** [comment]
@lavrd yeah, please merge it!

**Comment by lavrd** [comment]
I don't have rights to merge, maybe @kamikazechaser or @hibiken can help.

**Review by kamikazechaser — COMMENTED** [review]
Inline docs suggestion.

File: server.go [hunk 1]
```diff
@@ -104,6 +104,15 @@ type Config struct {
 	// If this is defined, then it MUST return a non-nil context
 	BaseContext func() context.Context
 
+	// TaskCheckInterval specifies the interval between checks for new tasks to process when all queues are empty.
```
[the remark this answers is not in the document]
Comment (krhubert) [reply server.go:107]: Thanks, updated!

**Review by kamikazechaser — APPROVED** [review]
LGTM

cc/ @hibiken

**Pull request merged**